src/mds/MDCache.cc

```diff
@@ -13488,6 +13488,13 @@ bool MDCache::dump_inode(Formatter *f, uint64_t number) {
   return true;
 }
 
+bool MDCache::dump_dir(Formatter *f, CDir *dir) {
+  f->open_object_section("dir");
+  dir->dump(f);
```

Comment:
This, by default, does not dump dentry items. Should that be made optional?

Reply:
> This, by default, does not dump dentry items. Should that be made optional?

Yes, I think it's a good idea and I have added an option for it. Thanks!